$NetBSD: patch-af,v 1.1 2000/10/07 19:24:15 kim Exp $

We don't need rps_dist.  We could build it if we always enable
threads and fix the Makefile to get CPPFLAGS and CFLAGS right.

--- programs/Makefile.orig	Mon Aug 21 14:08:05 2000
+++ programs/Makefile	Fri Oct  6 21:13:15 2000
@@ -8,7 +8,7 @@
 INSTALL_DIRS= IRRd irr_submit irr_rpsl_check irr_notify 
 #DIRS=$(PROGRAM_DIRS)
 DIRS=atomic_ops IRRd hdr_comm pgp irr_rpsl_check irrd_ops \
-	irr_notify irr_submit rps_dist
+	irr_notify irr_submit # rps_dist
 
 all:  make-programs
 
